functions/src/create-user.ts

```diff
@@ -69,6 +69,7 @@ export const createuser = newEndpoint(opts, async (req, auth) => {
     followerCountCached: 0,
     followedCategories: DEFAULT_CATEGORIES,
     shouldShowWelcome: true,
+    correctResolutionPercentageCached: 1,
```

Comment:
Maybe it should be a fraction instead of a percentage if it's from 0 to 1.
e.g. `correctResolutionFractionCached` or `fractionResolvedCorrectlyCached`.

I also think it would be ok to omit the Cached suffix: `fractionResolvedCorrectly`.